teps/0053-tekton-catalog-pipeline-organization.md

```diff
@@ -0,0 +1,477 @@
+
+## Motivation
+
+As the Tekton Catalog grows, it is important to define an organisation for pipeline as did for task in [TEP-0003: Tekton Catalog Organization](https://github.com/tektoncd/community/blob/main/teps/0003-tekton-catalog-organization.md), a set of rules and clear ownership that make catalog successful and sane in the long-term. This should also help other projects to define their own catalog that would satisfy the catalog contract.
+
+### Goals
+
+<!--
+List the specific goals of the TEP.  What is it trying to achieve?  How will we
+know that this has succeeded?
+-->
+
+### Non-Goals
+
+<!--
+What is out of scope for this TEP?  Listing non-goals helps to focus discussion
+and make progress.
+-->
+
+### Use Cases (optional)
+
+<!--
+Describe the concrete improvement specific groups of users will see if the
+Motivations in this doc result in a fix or feature.
+
+Consider both the user's role (are they a Task author? Catalog Task user?
+Cluster Admin? etc...) and experience (what workflows or actions are enhanced
+if this problem is solved?).
+-->
+
+## Requirements
+
+### Quality
+
+1. Resources should be well documented:
+
+   1. All configuration options are documented
+
+   2. Examples are provided
+
+2. If a resource is in the catalog, a user should feel confident that
+   it will work as advertised See [Tekton Catalog Test Infrastructure for
+   this](https://docs.google.com/document/d/1-czjvjfpuIqYKsfkvZ5RxIbtoFNLTEtOxaZB71Aehdg/edit#)
+
+3. When resources have parameters, resource or results, it must be
+   clear what values are allowed and the user must receive feedback
+   when they provide values that are not valid
+
+## Proposal
+
+### Support Tiers
+
+This proposal includes [three support
+levels](https://github.com/tektoncd/catalog/issues/5):
+
+1. **Community** - These are resources submitted by the community
+   which do not include [a
+   test](https://docs.google.com/document/d/1pZY7ROLuW47ymZYqUgAbxskmirrmDg2dd8VPATYXrxI/edit#heading=h.e9bh8qes6kh6) that can be tested successfully.
+
+   1. Not tested automatically
+
+   2. Not maintained by the tekton maintainers
+
+2. **Verified** - These are resources that include [a test](https://docs.google.com/document/d/1pZY7ROLuW47ymZYqUgAbxskmirrmDg2dd8VPATYXrxI/edit#heading=h.e9bh8qes6kh6)
+   that can be tested successfully but are owned by Tekton org members
+   outside of the OWNERS of the catalog.
+
+   3. Tested automatically
+
+   4. Not maintained by the tekton maintainers
+
+3. **Official** - These resources are verified but also are owned and
+   maintained by the Tekton catalog OWNERS, including being updated
+   [to be compatible with new versions of
+   Tekton.](https://docs.google.com/document/d/1pZY7ROLuW47ymZYqUgAbxskmirrmDg2dd8VPATYXrxI/edit#heading=h.t6fnduaxxkab)
+
+   5. Tested automatically
+
+   6. Maintained by the tekton maintainers
+
+   7. Image used are scanned periodically for CVE
+
+Those support tiers are mainly aimed for the Tekton Hub.
```

Comment:
i wonder if we really need to repeat these - maybe we can just link back to https://github.com/tektoncd/community/blob/main/teps/0003-tekton-catalog-organization.md#support-tiers for now? (i think eventually we'll want a separate TEP about the support levels in the catalog overall)